hack/make-rules/test-e2e-node.sh

```diff
@@ -28,6 +28,7 @@ skip=${SKIP-"\[Flaky\]|\[Slow\]|\[Serial\]"}
 parallelism=${PARALLELISM:-8}
 artifacts=${ARTIFACTS:-"/tmp/_artifacts/`date +%y%m%dT%H%M%S`"}
 remote=${REMOTE:-"false"}
+runtime=${RUNTIME:-"docker"}
```

Comment:
add this to the `Makefile`'s examples https://github.com/kubernetes/kubernetes/blob/master/Makefile#L200 ?